src/app/public/modules/lookup/types/lookup-add-click-event.ts

```diff
@@ -0,0 +1,3 @@
+export interface SkyLookupAddMoreClickedEvent {
```

Comment:
For the name of this interface, I prefer it to be `<NameOfComponent><NameOfEvent>EventArgs`, so in this case, it would be `SkyLookupAddClickEventArgs`.

Reply:
Done